mmv1/products/compute/api.yaml

```diff
@@ -8432,6 +8432,16 @@ objects:
         description: |
           Specifies how to handle instances when a node in the group undergoes maintenance. Set to one of: DEFAULT, RESTART_IN_PLACE, or MIGRATE_WITHIN_NODE_GROUP. The default value is DEFAULT.
         default_value: DEFAULT
+      - !ruby/object:Api::Type::NestedObject
+        name: 'maintenanceWindow'
```

Comment:
Will this be populated by the API if we don't send anything? Just thinking it might need to be marked computed to prevent diffs, but it depends on how the API handles it. Do we have a test where this field is unspecified, and if not can you add one?

Reply:
We have the [following example](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/templates/terraform/examples/node_group_basic.tf.erb) already existing where this parameter is not set, although to verify this window isn't set when maintenance_policy is set I'll run a local test.

Reply:
confirmed if not set, not returned